extensions/amp-sidebar/1.0/amp-sidebar-1.0.md

```diff
@@ -58,103 +62,240 @@ limitations under the License.
+### Opening and closing the sidebar
 
-Alternatively pressing the escape key on the keyboard will also close the sidebar.
+To toggle, open, or close the sidebar when an element is tapped or clicked, set the [`on`](../../../spec/amp-actions-and-events.md) action attribute on the element, and specify one of the following action methods:
+
+<table>
+  <tr>
+    <th>Action</th>
+    <th>Description</th>
+  </tr>
+  <tr>
+    <td>open (default)</td>
+    <td>Opens the sidebar</td>
+  </tr>
+  <tr>
+    <td>close</td>
+    <td>Closes the sidebar</td>
+  </tr>
+  <tr>
+    <td>toggle</td>
+    <td>Toggles the sidebar state</td>
+  </tr>
+</table>
+
+If the user taps back on the partially-visible main content area, this closes the sidebar.
+
+Alternatively, pressing the escape key on the keyboard will also close the sidebar.
 
 Example:
+
 ```html
 <button class="hamburger" on='tap:sidebar1.toggle'></button>
 <button on='tap:sidebar1'>Open</button>
 <button on='tap:sidebar1.open'>Open</button>
 <button on='tap:sidebar1.close'>x</button>
 ```
 
-## Attributes
+### Toolbar
 
-**side**
+You can create a toolbar element that displays in the `<body>` by specifying the toolbar attribute with a media query on a `<nav>` element that is a child of  `<amp-sidebar>`. The toolbar duplicates the `<nav>` element and its children.
 
-The `side` attribute may be set to `left` or `right` depending upon whether sidebar should open in the left or right side of the page. If a `side` is not set on the `amp-sidebar` then it will be inherited from the `body` tag's `dir` attribute (`ltr` => `left` , `rtl` => `right`) and if one does not exist then the `side` is defaulted to `left`.
+#### Behavior
 
-**open**
+- The sidebar may implement toolbars by adding nav elements with the toolbar attribute.
+- The nav element must be a child of `<amp-sidebar>` and follow this format: `<nav toolbar="(media-query)">`.
+    - For instance, this would be a valid use of toolbar: `<nav toolbar="(max-width: 1024px)">`.
+- The nav containing the toolbar attribute must only contain a single `<ul>` element, that contains `<li>` elements.
+    - The `<li>` elements may may contain any valid HTML elements (supported by AMP), or any of the AMP elements that `<amp-sidebar>` supports.
+- The nav element, or it's `<ul>`'s `<li>` elements, may also contain the attribute `toolbar-only`.
+    - The attribute `toolbar-only` will hide the elements with the attribute in the sidebar, but leave them shown in the toolbar.
+- The nav element may also contain the `target` attribute that accepts an element id.
+    - The `target` attribute will place the toolbar into the specified id of the element on the page, without applying the default toolbar class.
+    - For example: `<nav toolbar="(max-width: 1024px)" target="toolbar-target">`.
+- Toolbar behavior is only applied while the media-query is valid.
 
-The `open` attribute is present on the sidebar when it is open.
+Example: Basic Toolbar
```

Comment:
nit: Can you italicize this, it looks better when rendered in the docs. So, *Example: Basic Toolbar*.  Also for the rest of the "Example" subheadings. Thx.